web/src/api/hfiCalculatorAPI.ts

```diff
@@ -16,6 +16,7 @@ export interface StationDaily {
   bui: number
   fwi: number
   danger_class: number
+  ros: number
```

Comment:
Same comment as above - re. acronyms
```suggestion
  rate_of_spread: number
```